### DISCUSSION TITLE
Dudas con el TP 0

### DISCUSSION THREAD

**Discussion by Lautymendez** [INGCOM-UNRN-P1/cursada-2023#10] [Preguntas y respuestas] [answered]
Buenas, realizando el TP hice todo los pasos previos (instalacion de mingw y visual code) y al momento de poner el comando "gcc --version" en la terminal en Visual Studio Code me aparece este error:
```
$> : El término '$>' no se reconoce como nombre de un cmdlet, función, archivo de script o programa ejecutable. Compruebe si escribió correctamente el nombre o, si 
incluyó una ruta de acceso, compruebe que dicha ruta es correcta e inténtelo de nuevo.
En línea: 1 Carácter: 1
+ $> gcc --version
+ ~~
    + CategoryInfo          : ObjectNotFound: ($>:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException  
```

Si alguien me podria ayudar porfa se lo agradeceria

**Comment by martinvilu** [comment]
¿Como instalaste el vscode y el mingw?

**Reply by Lautymendez** [reply]
El vscode lo instale por este link : https://code.visualstudio.com/download ; y el mingw me di cuenta que lo instale desde otro link, ahi ya lo estoy descargando en el enlace que dejo en la discussion anterior. Ahi me fijo si funciona, si no, le contesto de vuelta

**Reply by Lautymendez** [reply]
Ahi me funciono, el error estaba que no habia hecho bien la instalacion del mingw. Gracias!

**Reply by martinvilu** [reply]
La instalación en sí no importa, solo si está configurado el `path` [wiki](https://github.com/INGCOM-UNRN-P1/cursada-2023/wiki/Instalaci%C3%B3n-de-las-herramientas)

Si está resuelto, marcalo como tal, así lo cerramo.

**Reply by Lautymendez** [reply]
Sisi perdon me exprese mal, osea lo configure en el path y todo. Ya probe abrir un cmd, poner gcc --version y me sale el mensaje correcto.

**Reply by martinvilu** [reply, marked as answer]
Bueno, ahora marcá como resuelto asi lo cerramo.
